webapp/pages/post/_id/_slug/index.vue

```diff
@@ -90,7 +90,17 @@
           @toggleNewCommentForm="toggleNewCommentForm"
         />
         <ds-space margin-bottom="large" />
-        <hc-comment-form v-if="showNewCommentForm" :post="post" @createComment="createComment" />
+        <comment-form
+          v-if="showNewCommentForm && !post.author.blocked"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-20T16:36:07Z" title="Monday, January 20th 2020, 5:36:07 pm +01:00">Jan 20, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -89,7 +89,19 @@
           @toggleNewCommentForm="toggleNewCommentForm"
         />
         <ds-space margin-bottom="large" />
-        <hc-comment-form v-if="showNewCommentForm" :post="post" @createComment="createComment" />
+        <comment-form
+          v-if="showNewCommentForm && !this.blocked"
```


```suggestion
          v-if="showNewCommentForm && !post.author.isBlocked"
```

I think that's enough and you can remove `blockedByPostAuthor`.